source/unified-test-format/unified-test-format.rst

```diff
@@ -1251,6 +1280,67 @@ expectedCmapEvent
   MUST be an empty document as all fields in this event are non-deterministic.
 
 
+expectedSdamEvent
+`````````````````
+
+The structure of this object is as follows:
+
+.. _expectedEvent_serverDescriptionChangedEvent:
+
+- ``serverDescriptionChangedEvent``: Optional object. Assertions for one or more
+  `ServerDescriptionChangedEvent <../server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events>`__ fields.
+
+  The structure of this object is as follows:
+
+  - ``previousDescription``: Optional object. A value corresponding to the server
+    description as it was before the change that triggered this event.  Test
+    runners MUST follow the rules in `Evaluating Matches`_ when processing this
+    assertion.
+
+  - ``newDescription``: Optional object. A value corresponding to the server
+    description as it was after the change that triggered this event. Test
+    runners MUST follow the rules in `Evaluating Matches`_ when processing this
+    assertion.
+
+  The structure of a server description object (which the ``previousDescription``
```

Comment:
the only field that is actually used in here is `type`, but I figured the rest of them might be useful in the future, so it was best to define it now so that we don't need to bump the format twice.

Reply:
I have concerns about generally applying "Evaluating Matches" to the entire ServerDescription object, since it's inconsistent with what we do for other special types (like APM events). I'd rather see it selectively apply to matching of specific fields as needed.

If `type` is really all we need to assert for these tests, I'd lead with just that and avoid the extra complexity until a test actually needs it. That seems consistent with your other decision to avoid introducing _all_ of the SDAM events up front. I'll note that we've also omitted fields on APM and CMAP events that we don't care to assert in unified tests.